cppwinrt/component_writers.h

```diff
@@ -860,7 +860,17 @@ catch (...) { return winrt::to_hresult(); }
         {
             auto format = R"(
 #if defined(WINRT_FORCE_INCLUDE_%_XAML_G_H) || __has_include("%.xaml.g.h")
+
 #include "%.xaml.g.h"
+namespace winrt::@::implementation
+{
+    template <typename D, typename... Args>
+    auto initialize_instance(%T<D, Args...>& instance)
+    {
+        static_cast<D&>(instance).InitializeComponent();
```

Comment:
Clearly this is in support of Xaml. Do we really want to advertise this for other uses?

Reply:
Not necessarily - I could be persuaded to remove the general use.